.gitignore

```diff
@@ -1,6 +1,9 @@
 # Composer
-vendor/*
 composer.lock
+vendor/*
 
 # PHPStorm
 .idea
+
+# PHPUnit
+.phpunit*
```

Comment:
this will ignore the phpunit.xml.dist that is already commited

Reply:
Good catch. Fixed.